content/ko/docs/topics/advanced.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "고급 헬름 기법"
+description: "헬름의 숙련된 사용자를 위한 다양한 고급 기능을 설명한다."
+weight: 9
+---
+
+이 섹션에서는 헬름 사용에 관한 다양한 고급 기능과 기법을 설명한다. 
+이 섹션의 정보는 차트 및 릴리스의 고급 사용자 정의 및 조작을 
+수행하려는 헬름의 "숙련 사용자"를 위한 것이다. 이러한 각 
+고급 기능에는 고유한 장단점 및 주의사항이 있으므로, 각 기능은 
+헬름에 대해 충분한 지식을 가지고 신중하게 사용해야한다. 
+바꿔 말해서, [Peter Parker 
+원칙](https://en.wikipedia.org/wiki/With_great_power_comes_great_responsibility)을 유념하자.
+
+## 포스트 렌더링
+포스트 렌더링은 차트 설치 프로그램이 헬름으로 설치하기 
+전에 렌더링된 매니페스트를 수동으로 
+조작, 구성 및 유효성 검사를 할 수 있는 기능을 
+제공한다. 이를 통해 [`kustomize`](https://kustomize.io)와 
+같은 도구를 필요로 하는 고급 구성을 이용하는 사용자들이, 
+공개 차트를 포크하거나 차트 관리자가 소프트웨어에 대한 
+모든 최종 구성 옵션을 지정하도록 요구하지 않고도 구성 
+변경 사항을 적용할 수 있도록 해준다.
```

Comment:
Do we need to add https://github.com/helm/helm-www/blame/master/content/en/docs/topics/advanced.md#L22-L24 ?
> There are also use cases for injecting common tools and side cars in enterprise environments or analysis of the manifests before deployment.

Reply:
Okay I will add this line.